cvmfs/s3fanout.cc

```diff
@@ -601,15 +616,15 @@ bool S3FanoutManager::MkV4Authz(const JobInfo &info, vector<string> *headers)
     scope + "\n" +
     hash_request;
 
-  string signing_key = GetAwsV4SigningKey(info, date);
+  string signing_key = GetAwsV4SigningKey(date);
```

Comment:
We should cache the signing key, I guess.  Just something to keep in mind.

Reply:
There is some sort of caching inside the function (the signing_keys_ map), but I think we should go in the opposite direction. Since time moves only forward, it only makes sense to cache these signing keys within one day. I suggest caching just one signing key, which gets overwritten at midnight every day.